NoSQL/ce-sec/Dockerfile

```diff
@@ -20,6 +20,18 @@ ENV KV_SERVICERANGE 5021-5049
 # specific to secure configuration
 RUN microdnf -y install openssl-1.1.1k && microdnf clean all
 
+# hadolint ignore=DL4006
+RUN useradd -d /kvroot  -m -s /bin/bash -u 1000 nosql && \
```

Comment:
I recommend using the username `oracle` instead of `nosql`.

Reply:
@mikebrey what is the decision here? (oracle or nosql_user) - see above Avi's recommendation

Reply:
I have added nosql-user as discussed @mikebrey. Do I need to change to oracle?

Reply:
Pick the one that would be less surprising to your users. I recommended `oracle` because that's the OUI default used by all Oracle Database and Middleware installs. Everyone knows `oracle:oinstall`, right? But if NoSQL has a different convention, use that.

Reply:
we generally do not use 'oracle' as our user name in nosql.     nosql-user works better for us.